src/antsibull_core/galaxy.py

```diff
@@ -356,6 +361,8 @@ async def get_latest_matching_version(
         spec = semver.SimpleSpec(version_spec)
         prereleases = []
```

Comment:
@felixfontein not really related to the PR but I saw this block of code and wanted to mention that I've implemented the "allow pre-release if no stables on galaxy" that's coming in core 2.16 (and likely 2.15 too): https://github.com/ansible/ansible/commit/7662a050853b41189a66c9a840091e9b05bcba49#diff-fe795e4ef8c29c07418d68d6a5c81d0d1779205a01b5b41a2f708d5184e15e2aR299. In case this'd be useful for antsibull...

Reply:
Nice! It won't be useful for antsibull though (or at least not anytime soon) since it doesn't use ansible-core's galaxy code or collection version resolving code at all.